platform/ios/src/MGLMapView.mm

```diff
@@ -3563,7 +3554,7 @@ - (void)_setVisibleCoordinates:(const CLLocationCoordinate2D *)coordinates count
     }
     
     MGLMapCamera *camera = [self cameraForCameraOptions:cameraOptions];
-    if ([self.camera isEqualToMapCamera:camera])
+    if ([self.camera isEqualToMapCamera:camera] && UIEdgeInsetsEqualToEdgeInsets(_contentInset, insets))
```

Comment:
`insets` doesn’t include any content inset. `padding` is the sum of `insets` and the content inset.

Reply:
→ #14790

Reply:
Nice catch. Thanks.
-> #14795